gallery/src/components/NxStatefulToggle/NxStatefulToggleExample.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { NxStatefulToggle } from '@sonatype/react-shared-components';
+
+function NxStatefulToggleExample() {
+  const onChange1 = (checkedState: boolean) => { alert(checkedState); },
+      onChange2 = (checkedState: boolean) => { alert(checkedState); };
+
+  return (
+    <div>
```

Comment:
No need for this div, I'd use a fragment

Reply:
> No need for this div, I'd use a fragment

https://github.com/sonatype/sonatype-react-shared-components/pull/250/commits/dc33d83ebc6c1322c4c8d41d8e8b9d19003d66e0